recipes/seq/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir, replace_in_file
+from conan.tools.scm import Version
+from conan.tools.env import VirtualBuildEnv
+import os
+
+required_conan_version = ">=2.1"
+
+class PackageConan(ConanFile):
+    name = "seq"
+    description = "A collection of original C++14 STL-like containers and related tools"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Thermadiag/seq"
+    topics = ("formatting", "hashmap", "radix", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    short_paths = True
+
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
```

Comment:
Note that this method is no longer useful in Conan 2 (as we always have cppstd!), and that specific incompatibilities with old compilers should be expressed directly in the validate method.

So for example, if `gcc 5` supports the min_cppstd, but it has a bug that prevents it from compiling this library until gcc 6, then just add that in the validate method, like

```
def validate(self)
    check_min_cppstd(self, 14)
    if self.settings.compiler == "gcc" and Version(self.settings.compiler.version) < "6":
        raise ConanInvalidConfiguration("Specific reason for each one")
```

instead of relying on a general method, this way the user will have more info :)

Thanks!

Reply:
Thank you!
Fixed.